<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPercutaneousImageGuidedDICOMWidget</class>
 <widget class="qSlicerWidget" name="qSlicerPercutaneousImageGuidedDICOMWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>AETitle / Port : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DICOMAETitleBox">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>SPLSNR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DICOMPortBox">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>99000; </string>
       </property>
       <property name="text">
        <string>1040</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DICOM Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DICOMDirectoryLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkDirectoryButton" name="DICOMListenerButton">
     <property name="text">
      <string>Start DICOM Listener</string>
     </property>
     <property name="options">
      <set>ctkDirectoryButton::HideNameFilterDetails|ctkDirectoryButton::ShowDirsOnly</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="DICOMManagerButton">
     <property name="text">
      <string>Open DICOM Manager</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>152</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
